CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+content for our official Technical Blog. Second, in order to make contributing
+a pleasant experience while maintaining a visual and consistent content standard,
+complete these steps before writing and submitting a blog entry for publishing.
+
+For ideas of topics to blog about, check out the end of this file!
+
+#### Preparing for your blog entry
+
+1. Fork this repo (https://github.com/rackerlabs/technical-blog), then
+   clone your fork.
+
+2. Before you start working, make sure your local content is up-to-date and
+   merged with the right upstream branch:
+
+      ```bash
+         git remote add upstream git@github.com:rackerlabs/docs-developer-blog.git
+         git checkout master
+         git fetch upstream
+         git checkout -b name-of-your-branch
+         git merge upstream/master
+      ```
+
+3. Create a new folder inside the `/hugo/content/blog` directory with the following naming
+   convention: `YYYY-MM-DD-title-of-your-post`, where `YYYY-MM-DD` is the
+   date you want your entry to be published.
+
+4. Within the newly created folder, create a new file with the following naming convention:
+   `index.md`.The published URL for your post becomes `https://docs.rackspace.com/blog/title-of-your-post/` when published.
+
+**Note:** If your post has images, you should place the image files within the newly created
+folder.
+
+#### Formatting your post
+
+The post should contain front-matter, an excerpt, and the actual content.
+
+##### Front-matter
+
+Add Jekyll front-matter (or metadata) to the top of the file you created in
+the previous step. for example:
+
+```
+---
+layout: post
+title: "Blog entry title"
+date: YYYY-MM-DD 23:59
+comments: true
+author: Author(s) name(s)
+published: true
+authorIsRacker: true
+#
+# The *authorAvatar* and *bio* entires are optional, but include them if you can!
+# The avatar must be a hosted image, such as a gravatar.
+#
+authorAvatar: 'https://www.gravatar.com/avatar/<insert hash for your headshot>'
+bio: "<insert a sentence or two about yourself in first or third person>"
+categories:
+    - This Category
+    - That Category
+    - Other Category
+#
+# Use canonical entry if you are republishing a blog from another site, such as
+# your personal blog.  Do  NOT republish without the autor's explicit permission.
+#
+canonical: https://original-url.link.com/post-name/
+metaTitle:
+metaDescription:
+ogTitle:
+ogDescription:
+#
+# The following properties are OPTIONAL and affect the text and image that
+# appear by default in link previews when sharing blog posts.
+#
+ogImage:
+twitterCreator: "@your_twitter_handle" # NOTE: The quotes are required!
+twitterDescription:
+twitterTitle:
+---
+```
+
+**NOTE:** The "ogImage" _must_ be a fully-qualified URL. If you'd like to use an
+image asset that is being uploaded as part of your blog post, the pattern for the
+final URL is as follows:
+
+  `<rackspace CDN>/<image name>-<sha256 hash>.<image extension>`
```

Comment:
I think we can cut this. Have authors put the image files in the page bundle. Let's get away from rackcdn complexity.

Reply:
This image affects what displays if you put a link to your blog in Slack, for example.  I doubt Slack knows how to link to an image file in a repo folder.